localization/ndt_scan_matcher/README.md

```diff
@@ -59,3 +64,11 @@ There are two main functions in this package:
 | `converged_param_transform_probability` | double | Threshold for deciding whether to trust the estimation result                                   |
 | `omp_neighborhood_search_method`        | int    | neighborhood search method in OMP (0=KDTREE, 1=DIRECT26, 2=DIRECT7, 3=DIRECT1)                  |
 | `omp_num_threads`                       | int    | Number of threads used for parallel computing                                                   |
+
+### Regularization Parameters
+
+Regularization is available only with NDT_OMP.
+| Name | Type | Description |
+| --------------------------------------- | ------ | ----------------------------------------------------------------------- |
+| `regularization_enabled` | bool | Flag to add regulalization term to NDT optimization (FALSE by default) |
```

Comment:
```suggestion
| `regularization_enabled` | bool | Flag to add regularization term to NDT optimization (FALSE by default) |
```

Reply:
Thenks for your suggestion.  I fixed it in [23ec0c9](https://github.com/autowarefoundation/autoware.universe/pull/1006/commits/23ec0c938c2551293602d6a97237a85dc66634d9)